client/src/components/ChallengeCard.tsx

```diff
@@ -0,0 +1,22 @@
+import Card from "react-bootstrap/Card";
+import Button from "./Button.tsx";
+
+type ChallengeCardProps = {
+  title: string;
+  description: string;
+  href: string;
+};
+
+function ChallengeCard({ title, description, href }: ChallengeCardProps) {
```

Comment:
please add the difficulty prop

Reply:
We want to display the difficulty for sure. In the future we might add `solved` status and other things, but don't worry about that for now. Just add the difficulty and maybe display it by adding color to the challenge title. 

Maybe try the bootstrap 
.text-success
.text-warning
.text-danger

for 

easy
medium
hard

Reply:
P.S. I am using an `enum` in my Challenge page. 

```js
enum ChallengeDifficulties {
    EASY,
    MEDIUM,
    HARD
}
```

Reply:
```js 
    function getDifficulty(difficulty: ChallengeDifficulties): string{
        switch(difficulty){
            case ChallengeDifficulties.EASY:
                return "var(--bs-teal)";
            case ChallengeDifficulties.MEDIUM:
                return "var(--bs-warning)";
            case ChallengeDifficulties.HARD:
                return "var(--bs-danger)";
        }
    }
```

Reply:
I went for having stars rather than using color.